stats/opentelemetry/opentelemetry.go

```diff
@@ -181,6 +205,8 @@ type attemptInfo struct {
 
 	pluginOptionLabels map[string]string // pluginOptionLabels to attach to metrics emitted
 	xdsLabels          map[string]string
+
+	ti *attemptTraceSpan
```

Comment:
Same. Change to attemptInfo. We should restrict to traces. attemptInfo can have traces, metrics etc. together as separate fields within